tests/unit/visualise/test_generate_map_based_visualisations.py

```diff
@@ -18,6 +19,12 @@ def AURN_filepath(sampledir):
     return AURN_filepath
 
 
+@pytest.fixture()
+def AURN_savepath(tmp_output_path):
+    AURN_savepath = os.path.join(tmp_output_path, "AURN.html")
+    return AURN_savepath
```

Comment:
your variable has the same name as the function its in. I think this works ok because the variable only (shadows)[https://en.wikipedia.org/wiki/Variable_shadowing] the function name within the scope of that function, but it's best to give it a different name

Reply:
Done!